Docs/user_guide/modules/markups.md

```diff
@@ -118,26 +164,42 @@ Control points section:
   - Name: A short name for this control point, displayed in the viewers as text next to the glyphs.
   - Description: A longer description for this control point, not displayed in the viewers.
   - X, Y, Z: The RAS coordinates of this control point, 3 places of precision are shown.
+  - State: The current state of the control point. Clicking on the current state will cycle through the possible states.
+    - Edit: The control point is currently being placed. Only one control point can be in the edit state at a time. If the state of another control point is set to edit, then the current control point state will be set to clear.
+    - Skip: The control point is not currently defined, and cannot be selected for placement.
+    - Restore: The control point has a defined position. Entering this state will restore the last known position of the control point.
+    - Clear: The control point has not yet been placed, and can be selected for placement.
 - Advanced section:
   - Move Up: Move a highlighted control point up one spot in the list.
   - Move Down: Move a highlighted control point down one spot in the list.
-  - Add Control Point: Add a new control point to the selected list, placing it at the origin
+  - Add Control Point: Add a new unplaced control point to the selected list, creating it with an undefined position.
   - Naming:
     - Name Format: Format for creating names of new control points, using sprintf format style. %N is replaced by the list name, %d by an integer.
     - Apply: Rename all control points in this list according to the current name format, trying to preserve numbers. A quick way to re-number all the control points according to their index is to use a name format with no number in it, rename, then add the number format specifier %d to the format and rename one more time.
     - Reset: Reset the name format field to the default value, %N-%d.
+  - Convert annotation fiducials: Uses annotation fiducial hierarchies to convert them to markups. Removes the annotation nodes once completed.
 
-Measurements section:
+### Measurements Section:
 - This section lists the available measurements of the selected markup
   - `length` for line and curve
   - `angle` for angle markups
   - `curvature mean` and `curvature max` for curve markups
+  - `area` for plane markups
+  - `volume` for ROI markups
 - In the table below the measurement descriptions, the measurements can be enabled/disabled
   - Basic measurements (e.g. length, angle) are enabled by default
   - Curve markups support curvature calculation, which is off by default
     - When turned on, the curvature data can be displayed as scalar coloring (see Display/Scalars above)
 
-Curve settings section:
+### Export/Import Table Section:
```

Comment:
```suggestion
### Export/Import Table section
```

Reply:
@jcfr Was the lowercase suggestion for "section" intentional here?

Reply:
Indeed, I updated the header so that there are all consistent:
- without `:`
- with lower case `s` for `section`